### PR TITLE
trigger pipeline that updates grobid and grobid trainer tag

### PR DESCRIPTION
**Pull request by tayowonibi**
partly resolves[ #5490](https://github.com/elifesciences/issues/issues/5490) - triggers pipeline that update grobid and grobid trainer tags

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -65,6 +65,12 @@ elifePipeline {
                     return m
                 })
             }
+
+            stage 'Trigger grobid and grobid trainer tag update', {
+                def grobidTag = allGrobidTags.last()
+                def grobidTrainerTag = "${grobidTag}-${commit}"
+                build job: 'dependencies/dependencies-sciencebeam-trainer-delft-update-grobid-and-trainer',  wait: false, parameters: [string(name: 'grobid_tag', value: grobidTag), string(name: 'grobid_trainer_tag', value: grobidTrainerTag)]
```
Comment (de-code) [inline Jenkinsfile]: very minor: just wondering whether we can break that line up (in terms of line length)

**Review by de-code — APPROVED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -65,6 +65,17 @@ elifePipeline {
                     return m
                 })
             }
+
+            stage 'Trigger grobid and grobid trainer tag update', {
+                def grobidTag = allGrobidTags.last()
+                def grobidTrainerTag = "${grobidTag}-${commit}"
+                build   job: 'dependencies/dependencies-sciencebeam-trainer-delft-update-grobid-and-trainer',
```
Comment (giorgiosironi) [inline Jenkinsfile]: I assume this runs from https://alfred.elifesciences.org/job/libraries/job/library-sciencebeam-trainer-grobid/, so you need:
```
'../dependencies/dependencies-sciencebeam-trainer-delft-update-grobid-and-trainer'
```
as a relative path.

Comment (tayowonibi) [reply Jenkinsfile]: just a general unrelated question - what criteria determines which pipeline remain in root path and which goes into `image`, `dependencies`, `libraries`, `jobs` . .

Comment (giorgiosironi) [reply Jenkinsfile]: Probably only `test-*` and `prod-*` pipelines should stay at the top level, as we select them from the [build monitor](https://alfred.elifesciences.org/view/Build%20Monitor%20(projects)/). The assumption is that if a library fails is less important than a real deployment failing.

**Pull request merged**